addon/-task-instance.js

```diff
@@ -365,7 +383,6 @@ let taskInstanceAttrs = {
       return;
     } else {
       if (done && value === undefined) {
-        this.set('isFinished', true);
```

Comment:
This was extraneous even before this PR, as `this._finalize` will always set isFinished.